Sukanta_Roy/progress.md

```diff
@@ -19,58 +19,68 @@ Overall Direction:
 ---
 
 Week 1 Plan:
-- [ ] Work on [Instructor: edit rubric question: tweak UI to prevent misunderstanding of weights](https://github.com/TEAMMATES/teammates/issues/8824)
+- [x] Work on [Instructor: edit rubric question: tweak UI to prevent misunderstanding of weights](https://github.com/TEAMMATES/teammates/issues/8824)
 - [ ] Implement the backend part of [MCQ questions: support attaching weights to options](https://github.com/TEAMMATES/teammates/issues/2776)
 	- [ ] Update `QuestionDetails` class
 	- [ ] Update `Logic` class
 	- [ ] Update `Action` class
 	- [ ] Add Corrosponding Unit and Integration tests
 - [ ] Code review and fix
 
+Week 1 Work:
+- [x] Merged: [Instructor: edit rubric question: tweak UI to prevent misunderstanding of weights](https://github.com/TEAMMATES/teammates/issues/8824)
+
+Week 1 Retrospect:
+
+This week the plan was to complete the backend of MCQ weights feature, and I edited `FeedbackMcqQuestionDetails.java` accordingly, but
+I forgot to take into account about MCQ `other` option and how weight feature should behave in case of the `Other` option. After discussing with my mentors and Prof. Damith, we came to the conclusion that, at this point, we will add `weight` feature to `other` option too, giving the instructor full flexibility of the feature. We also discussed the best possible way to store the necessary fields in the datastore. In coming weeks, I will have to change my initial implementation of the weight feature to add the feature for the `Other` option. 
+
 Week 2 Plan:
 **Final Semester Exams**
 - [ ] Finalize the sample question list for the new question template
 
+Week 2 Work:
+- [x] Made 3 sample templates to choose from, for the `Add new template` feature.
```

Comment:
Include the link to the issue?

Reply:
Oops! forgot to include the link after creating the issue. Doing that.